docs/how-clerk-works/tokens-signatures.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: 'How Clerk Works: Tokens & Signatures'
+description: Learn about JWTs, digital signtures, and how Clerk uses them to securely authenticate users.
+---
+
+## Digital Signatures
+
+Digital signatures are a mathematical process that can be used to guarantee that a message sent from one person to another in public is genuine. In other words, it can guarantee for the recipient of a message that:
+
+1. The message definitely came from a specific sender (authenticity)
+1. The message’s contents were not modified from how they were written by the sender (integrity)
+
+It’s important to note that signatures do not actually encrypt a message - anyone can read the message’s contents.
+
+The way this works is as such:
+
+- Some fancy math is used to generate a pair of keys: a public key and a private key. This process is referred to as "public key cryptography".
```

Comment:
Nit: I get the point of "some fancy math", but maybe we use a slightly less throw away approach? I feel like there are _some_ who would read this as us being unserious, even though that's certainly not the case.

Reply:
or maybe we link "fancy math" to a legitimate deep dive? That would mitigate the concern IMO.

Reply:
```suggestion
- Some [fancy math](https://en.wikipedia.org/wiki/RSA_(cryptosystem)#Operation) is used to generate a pair of keys: a public key and a private key. This process is referred to as "public key cryptography".
```